### DISCUSSION TITLE
Tad 0.10.0

### DISCUSSION THREAD

**Discussion by antonycourtney** [antonycourtney/tad#164] [Announcements]
# Tad 0.10.0.

## New Features
- Tad now uses <a href="https://duckdb.org/">DuckDb</a> instead of SQLite, enabling much better load times and interactive performance, especially for large data files.
- Added direct support for Parquet and compressed CSV files.
- Added a new <i>Data Sources</i> sidebar showing available tables, data files and folders, and allowing fast switching between them.
- Tad can open DuckDb and SQLite Database files: <pre>$ tad myDatabase.duckdb</pre> or <pre>$ tad myDatabase.sqlite</pre>
- Ability to open filesystem directories directly for quick browsing and exploration of collections of tabular data files.

## Internal Changes

This release is a major restructuring and rewrite of the Tad internals:

- Implementation now structured as a <a href="https://lerna.js.org/">Lerna</a> monorepo, split into 12 sub-modules.
- Implementation ported to TypeScript and UI code updated to React Hooks
- Main pivot table React component (tadviewer) built as an independent module, enabling embedding in other applications</li>
- Experimental proof-of-concept packaging of Tad as a web app and a reference web server, illustrating how Tad could be deployed on the web.
- Experimental proof-of-concept support for other database backends (Snowflake, BigQuery, AWS Athena)

<hr /><em>This discussion was created from the release <a href='https://github.com/antonycourtney/tad/releases/tag/v0.10.0'>Tad 0.10.0</a>.</em>

**Comment by jzabroski** [comment]
Could you use any help going forward?
